src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dns/ClusterDnsCache.scala

```diff
@@ -21,16 +20,6 @@ object ClusterDnsCache {
 
   @volatile var HostToIp: Map[Host, IP] = Map.empty
 
-  def props(proxyConfig: ProxyConfig, dbRef: DbReference): Props =
-    Props(new ClusterDnsCache(proxyConfig, dbRef))
-
-  // Actor messages:
-  sealed trait ClusterDnsCacheMessage
-  case object RefreshFromDatabase extends ClusterDnsCacheMessage
-  case class ProcessClusters(clusters: Seq[Cluster]) extends ClusterDnsCacheMessage
-  case class ProcessReadyCluster(cluster: Cluster) extends ClusterDnsCacheMessage
-  case class GetByProjectAndName(googleProject: GoogleProject, clusterName: ClusterName) extends ClusterDnsCacheMessage
-
   // Responses to GetByProjectAndName message
```

Comment:
Update the comment above since we don't have a `GetByProjectAndName` message anymore